coderd/database/queries/workspaceagentstats.sql

```diff
@@ -90,7 +90,26 @@ ORDER BY
 	date ASC;
 
 -- name: DeleteOldWorkspaceAgentStats :exec
-DELETE FROM workspace_agent_stats WHERE created_at < NOW() - INTERVAL '180 days';
+DELETE FROM
+	workspace_agent_stats
+WHERE
+	created_at < (
+		SELECT
+			COALESCE(
+				-- When generating initial template usage stats, all the
```

Comment:
This change may be little too soon to drop. If customers identify problems with the release, they will decide to roll back to the previous version. Boom! No `workspace_agent_stats` anymore.

Reply:
I agree, I made this a separate PR so we can defer it for later.

Reply:
Rollbacks require db backups at present. We just do not support downgrades at present. 

And not supported like, you can't even really do it. You have to run the `migration downgrade` with the latest version, then rollback. We have 0 documentation for this, so no customer should do it.